package.json

```diff
@@ -80,6 +80,7 @@
     "@angular/compiler": "~9.1.11",
     "@angular/core": "~9.1.11",
     "@angular/forms": "~9.1.11",
+    "@angular/localize": "^9.1.11",
```

Comment:
See my i18n comment

Reply:
@fkolar have removed localize library, .xlf files and modified the ICU expression in lib file. There is no point adding a whole new library just for one component's changes.